test/asyncJest.spec.js

```diff
@@ -10,13 +10,14 @@ a função recebe como parâmetro true e false, respectivamente.
 ATENÇÃO!!! Edite apenas este arquivo. Não altere os arquivos da pasta 'src'.
 */
 
-describe("o retorno do telefonema", () => {
-  test("atende", () => {
-    assert.fail();
-    // Insira seu teste assíncrono aqui
+describe('1 - O retorno do telefonema', () => {
+  test('atende', () => {
+    expect.assertions(1);
+    return expect(answerPhone(true)).resolves.toEqual('Oi!');
   });
-  test("ocupado", () => {
-    assert.fail();
-    // Insira seu teste assíncrono aqui
+
+  test('ocupado', () => {
+    expect.assertions(1);
+    return expect(answerPhone(false)).rejects.toEqual('Infelizmente não podemos atender...');
```

Comment:
```suggestion
    return expect(answerPhone(false)).rejects.toThrow('Infelizmente não podemos atender...');
```

Aqui ele lança um erro, aí o toEqual não rola